elements/image/README.md

```diff
@@ -0,0 +1,52 @@
+Images are one of the most important design elements of an commercial email, but they need to be used with caution. [More on that](https://blog.edmdesigner.com/12-extremely-easy-tips-for-effective-use-of-images-in-email-design/)
```

Comment:
an commertial -> a commertial

Reply:
I like that you linked our article! :)

Reply:
it is easier than write it down again :)